spec/requests/home_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe "Homes", type: :request do
+  describe "GET /" do
+    it "returns http success" do
+      get "/"
```

Comment:
You can also put the root path.

```suggestion
      get root_url
```

Reply:
I already made this change.